threshsign/src/ThresholdAccumulatorBase.cpp

```diff
@@ -23,6 +23,7 @@ void ThresholdAccumulatorBase<VerificationKey, NumType, SigShareParserFunc>::set
   assertNotNull(msg);
   assertStrictlyPositive(len);
 
```

Comment:
Please add comments in code

Reply:
this code will not be hit while generating the binary in mac